executor/list_nodes.go

```diff
@@ -56,15 +58,31 @@ func ListNodes(client *Client) error {
 		return errTable
 	}
 
-	for _, info := range listTableResp.Infos {
-		queryCfgResp, err := client.Meta.QueryConfig(ctx, info.AppName)
-		if err != nil {
-			return err
-		}
-		nodes, err = fillNodesInfo(nodes, queryCfgResp.Partitions)
-		if err != nil {
-			return err
+	err = func() error {
+		var mu sync.Mutex
+		var funcs []func() error
+
+		for _, info := range listTableResp.Infos {
+			info := info
+			funcs = append(funcs, func() error {
+				queryCfgResp, err := client.Meta.QueryConfig(ctx, info.AppName)
+				mu.Lock()
+				if err != nil {
+					return err
+				}
+				nodes, err = fillNodesInfo(nodes, queryCfgResp.Partitions)
+				if err != nil {
+					return err
+				}
+				mu.Unlock()
+				return nil
+			})
 		}
+		return batchErr.AggregateGoroutines(funcs...)
+	}()
```

Comment:
```suggestion
		var mu sync.Mutex
		var funcs []func() error

		for _, info := range listTableResp.Infos {
			info := info
			funcs = append(funcs, func() error {
				queryCfgResp, err := client.Meta.QueryConfig(ctx, info.AppName)
				mu.Lock()
				if err != nil {
					return err
				}
				nodes, err = fillNodesInfo(nodes, queryCfgResp.Partitions)
				if err != nil {
					return err
				}
				mu.Unlock()
				return nil
			})
		}
		err = batchErr.AggregateGoroutines(funcs...)
```

Do not unnecessarily wrap your code nested.